platforms/jvm/jacoco/src/integTest/groovy/org/gradle/testing/jacoco/plugins/JacocoPluginMultiVersionIntegrationTest.groovy

```diff
@@ -41,7 +41,7 @@ class JacocoPluginMultiVersionIntegrationTest extends JacocoMultiVersionIntegrat
         succeeds('test', 'jacocoTestReport')
 
         then:
-        file(REPORTING_BASE).listFiles().collect { it.name } as Set == ["jacoco", "tests"] as Set
+        file(REPORTING_BASE).listFiles().collect { it.name } as Set == ["jacoco", "tests", "aggregate-test-results"] as Set
```

Comment:
🤔 What should we do about this?  Is it expected that we get a new set of results now?  If so we should note this in the release notes/upgrade guide.